app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -71,10 +71,7 @@
             label: logo_link_title,
           }
         } do %>
-          <%= render "govuk_publishing_components/components/layout_for_public/header_logo", {
-            hide_logo_text: hide_logo_text,
-            logo_text: logo_text
-          } %>
+          <%= render "govuk_publishing_components/components/govuk_logo/govuk_logo" %>
```

Comment:
There might be one more occurrence of `hide_logo_text` that can be removed here:

https://github.com/alphagov/govuk_publishing_components/blob/e0595b1f9e774604f71a64d697df4f9e67bdfe4f/app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb#L8

Reply:
Thanks, I've removed this occurrence as well